indico/queries/storage.py

```diff
@@ -133,3 +141,103 @@ def process_response(self, uploaded_files: List[dict]) -> List[str]:
 
 # Alias to ensure backwards compatibility
 UploadImages = CreateStorageURLs
+
+
+class GetDownloadUrl(GraphQLRequest):
+    """
+    Retrieve a signed url with a uri
+
+    Args:
+        uri (str): the storage uri prefixed by indico-file:///storage
+    Returns:
+        url (str): signed url of storage object
+    """
+
+    query = """
+        mutation($uri: String!) {
+            requestStorageDownloadUrl(uri: $uri) {
+                signedUrl
+            }
+        }
+    """
+
+    def __init__(self, *, uri: str):
+        uri = "indico-file:///storage" + uri
+        super().__init__(self.query, variables={"uri": uri})
+
+    def process_response(self, response: dict) -> str:
+        response = super().process_response(response)
+        return response["requestStorageDownloadUrl"]["signedUrl"]
+
+
+class GetUploadURL(GraphQLRequest):
+    """
+    Receive a signed url for the file being uploaded
+
+    """
+
+    query = """
+        mutation {
+            requestStorageUploadUrl {
+                signedUrl
+                relativePath
+            }
+        }
+    """
+
+    def __init__(self):
+        super().__init__(self.query)
+
+    def process_response(self, response: dict) -> dict:
```

Comment:
If you could add types for this response, it would be great. It'll help clients with autocomplete, even if it's something simple.